Doc/library/profile.rst

```diff
@@ -80,10 +80,10 @@ the following::
         1    0.000    0.000    0.000    0.000 _compiler.py:598(_code)
         1    0.000    0.000    0.000    0.000 _parser.py:435(_parse_sub)
 
-The first line indicates that 214 calls were monitored.  Of those calls, 207
+The first line indicates that 214 calls were monitored. Of those calls, 207
```

Comment:
Let's not change unrelated whitespace.
```suggestion
The first line indicates that 214 calls were monitored.  Of those calls, 207
```

Reply:
Sure, I also thought it was unrelated, just pretty close to context too. Returned

Reply:
Thanks. I also prefer one space, but we allow either one or two: https://devguide.python.org/documentation/style-guide/#use-of-whitespace